### PR TITLE
0.1.6 changelog

### PR DESCRIPTION
**Pull request by jakecraige**
Using new format stolen from chai.

Snuck in a script for generating it as well ;)

Also, it may be nice to start a convention of having good titles with tags like BREAKING, BUGFIX, etc in the title so we get prettier output with less manual labor

### CODE REVIEW

**Comment by hhff** [comment]
woah this is rad

File: bin/changelog [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+var compareCommits = Promise.denodeify(github.repos.compareCommits);
+var currentVersion = 'v' + require('../package').version;
+
+compareCommits({
+  user: 'ember-cli',
+  repo: 'ember-cli',
+  base: currentVersion,
+  head: 'master'
+}).then(function(res) {
+  return res.commits.map(function(commitInfo) {
```
Comment (jakecraige) [inline bin/changelog]: All that functional programming I've been learning is rubbing off on me. `map.filter.map.sort.map...`

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,41 @@
 ### Addons
 ### Blueprints
 
+### 0.1.6
+
+The following changes are required if you are upgrading from the previous
+version:
+
+- Users
+  + Upgrade you project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
```
Comment (jayphelps) [inline CHANGELOG.md]: s/you/your

File: bin/changelog [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  console.error(err);
+})
+
+function generateChangelog(contributions) {
+  var header = multiline(function() {/*
+The following changes are required if you are upgrading from the previous
+version:
+
+- Users
+  + Upgrade you project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
```
Comment (jayphelps) [inline bin/changelog]: s/you/your

**Comment by Dhaulagiri** [comment]
Curious if it would make sense to continue including the steps on how to upgrade in each release's changelog?

**Comment by Dhaulagiri** [comment]
Nevermind, I missed the links to the docs in the changelog...

**Pull request merged**